pkgs/os-specific/linux/nmon/builder.sh

```diff
@@ -0,0 +1,11 @@
+source $stdenv/setup
```

Comment:
Nit: this is an old-school way of packaging; it's way more common to inline the builder in the Nix expression/rely on the standard build sequence. In this case, I think you can get away with setting `buildPhase` and `installPhase` (setting `buildCommand` could work too, but then you need more boilerplate).

Reply:
Do we still have ancient documentation, which show examples like this?

Reply:
this is nice for understanding, but there should be a warning sign or something: https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv.xml#L90